src/gt4py/next/program_processors/runners/dace_iterator/workflow.py

```diff
@@ -115,15 +116,39 @@ class Meta:
         model = DaCeTranslator
 
 
+class CompiledDaceProgram(stages.CompiledProgram):
+    sdfg_program: dace.CompiledSDFG
+    sdfg_arg_position: list[
```

Comment:
Totally optional:
It might be more elegant to use the type `list[str]` thus the position of the argument, i.e. its index, is directly mapped to its name.
Furthermore if you modify `arg_names` then you could use that instead.

Reply:
I am interested in reducinf the call overhead, here. Looking for the index of an argument, at each SDFG call, is too expensive.